python/ql/src/semmle/python/security/Paths.qll

```diff
@@ -3,16 +3,18 @@ import python
 import semmle.python.security.TaintTracking
 
 query predicate edges(TaintedNode fromnode, TaintedNode tonode) {
-    fromnode.getASuccessor() = tonode
+    fromnode.getASuccessor() = tonode and
+    /* Don't record flow past sinks */
+    not fromnode.isVulnerableSink()
```

Comment:
Does this mean that a piece of code like
```
x = TAINT_SOURCE()
y = SINK(x)
z = SINK(x)
```
will only contain an edge for `y`, and not for `z`?

Reply:
In most cases there won't be an edge to `y` or `z`, unless `SINK` is something like
```
def SINK(x):
    return x
```

What this does do, is remove the edges within `SINK` and anything that `SINK` calls.

Reply:
Makes sense. I have no further comments. I'll go ahead with the merge.